runtime/lib/mirrors_impl.dart

```diff
@@ -1676,4 +1681,20 @@ class _Mirrors {
     }
     return typeMirror;
   }
+
+  static Type _resolveType(Type key, List<Type> typeArguments) {
+    if (typeArguments.isEmpty) {
+      throw new ArgumentError.value(
+        typeArguments, 'typeArguments', 'Type arguments list can not be empty.'
```

Comment:
This doesn't look like normal indentation. The ");" should not be on a new line.